install.sh

```diff
@@ -2,21 +2,20 @@
 set -o errexit
 
 version=$(curl -sS https://update.tabnine.com/version)
-case $(uname -s) in
-    "Darwin")
-        platform="apple-darwin"
-        ;;
-    "Linux")
-        platform="unknown-linux-musl"
-        ;;
-esac
-triple="$(uname -m)-$platform"
 
-cd $(dirname $0)
-path=$version/$triple/TabNine
-if [ -f binaries/$path ]; then
-    exit
-fi
-echo Downloading version $version
-curl https://update.tabnine.com/$path --create-dirs -o binaries/$path
-chmod +x binaries/$path
+targets='i686-unknown-linux-musl
+    x86_64-apple-darwin
+    aarch64-apple-darwin
+    x86_64-unknown-linux-musl'
+
+echo "$targets" | while read target
```

Comment:
I don't like how this downloads every version of TabNine...is changing this file actually necessary? I think it would download aarch64-apple-darwin on M1 without any changes.

Reply:
Because cannot detect Vim binary type in shell script.

Reply:
It shouldn't matter though? x86 processes can fork arm64 processes and vice versa.

Reply:
I have added platform check.

Reply:
It seems unnecessary to download multiple versions on Linux. Could you change that back to just uname -m?

Reply:
OK. I have fixed it.